### ISSUE TITLE
TypeError: next is not a function

### ISSUE THREAD

**Issue by azhar294** [feathers-plus/validate-joi#13]
```
error: TypeError: next is not a function
at /home/myproject/backend-api/node_modules/feathers-hooks-validate-joi/index.js:26:9
at internals.Object._validateWithOptions (/home/myproject/backend-api/node_modules/joi/lib/types/any/index.js:654:20)
at module.exports.internals.Any.root.validate (/home/myproject/backend-api/node_modules/joi/lib/index.js:121:23)
at Object.validatorInner (/home/myproject/backend-api/node_modules/feathers-hooks-validate-joi/index.js:14:9)
at current.then (/home/myproject/backend-api/node_modules/@feathersjs/commons/lib/hooks.js:115:46)
at process._tickCallback (internal/process/next_tick.js:68:7)
```
node@10.16.3
feathers-hooks-validate-joi@2.0.0

**Comment by gabrielstuff** [comment]
Encounter same issue. With node 8.9.1 and feathers-hooks-validate-joi@2.0.0 or lower version.

**Comment by gabrielstuff** [comment]
Ok, the plugin need some update to work with `feathers v4`.

So basically to use it you need : 

```
"@feathersjs/configuration": "^1.0.1",
"@feathersjs/errors": "3.3.0",
"@feathersjs/express": "1.2.3",
"@feathersjs/feathers": "3.1.7",
"@feathersjs/socketio": "3.2.2",
```

Make sure to clean your folder node_modules/ and start with a fresh `npm i`

**Comment by marshallswain** [comment]
Fixed in `@feathers-plus/validate-joi@3.0.0`.
